### PR TITLE
Branch change app colour

### PR DESCRIPTION
**Pull request by delishad21**
Currently, the application is a plain dark mode colour scheme. In a world where AIs like Devin threaten to take our jobs. Humanising touches to applications such as theming and design choices are needed for us to keep our jobs. Lets change the colour scheme of the application so it matches with the main platform of the course that this application is created for.

### CODE REVIEW

**Review by Zer0Legion — COMMENTED** [review]
I love the new colours! It appropriately reflects the app as having a closer theme to Avengers and Source Academy in CS1101S. Truly brings back nostalgic memories when I was taking CS1101S both as an Avenger and as a student. The new colours definitely make me want to be a head TA for CS1101S.

File: src/main/resources/view/DarkTheme.css
Comment (jayllo-c) [inline src/main/resources/view/DarkTheme.css:1]: I love how the colours now match source academy!

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -40,9 +40,9 @@
 }
 
 .table-view {
-    -fx-base: #1d1d1d;
-    -fx-control-inner-background: #1d1d1d;
-    -fx-background-color: #1d1d1d;
+    -fx-base: #1d2935;
```
Comment (Pughal77) [inline src/main/resources/view/DarkTheme.css:43]: I like this color. Good theme follows Source Academy

**Comment by danielleloh** [comment]
I think its a good idea to match the colour to source academy to make it more relevant to the context of the project

**Review by Pughal77 — COMMENTED** [review]
Nice

**Review by jayllo-c — COMMENTED** [review]
Would be a nice experience for head TAs!

**Pull request merged**